services/blockchain-indexer/shared/dataService/business/interoperability/blockchainApps.js

```diff
@@ -71,11 +75,30 @@ const getBlockchainApps = async (params) => {
 
 	const total = await blockchainAppsTable.count(params);
 
-	blockchainAppsInfo.data = await blockchainAppsTable.find(
+	const resultSet = await blockchainAppsTable.find(
 		{ ...params, limit: params.limit || total },
 		Object.getOwnPropertyNames(blockchainAppsTableSchema.schema),
 	);
 
+	const { chainID } = (await getNetworkStatus()).data;
```

Comment:
Usual alternative way of writing.

```suggestion
	const { data: chainID } = await getNetworkStatus();
```

Reply:
If refactoring, then it must be like this:
```
const { data: { chainID } } = await getNetworkStatus();
```